services/gateway/passThrough/passThrough.go

```diff
@@ -20,12 +23,16 @@ func Mount(upstreams *appConfig.Upstreams, v1Hosts *appConfig.V1Hosts, middlewar
 	apiForwarder := proxy.New(api, nil)
 	authoringForwarder := proxy.New(authoring, nil)
 
+	// metrics
+	apiMetricsMiddleware := metrics.NewMetricsMiddleware("api", metricsVar)
+	authoringMetricsMiddleware := metrics.NewMetricsMiddleware("authoring", metricsVar)
+
 	// Mounting handlers
 	for _, host := range v1Hosts.API {
-		router.Host(host).Handler(middleware.With(apiForwarder))
+		router.Host(host).Handler(middleware.With(apiMetricsMiddleware).With(apiForwarder))
```

Comment:
You can use it like middleware .With(apiMetricsMiddleware, apiForwarder), aren't you?